ast/term.go

```diff
@@ -2098,14 +2119,14 @@ func (obj *object) Filter(filter Object) (Object, error) {
 
 // Len returns the number of elements in the object.
 func (obj object) Len() int {
-	return len(obj.keys)
+	return len(obj.sortedKeys())
```

Comment:
Same here: sorted or not, the length should be the same.

Reply:
Yep, will patch. :+1: